argilla-server/src/argilla_server/telemetry/_client.py

```diff
@@ -98,7 +101,18 @@ async def track_api_request(self, request: Request, response: Response) -> None:
             if argilla_error := get_request_error(request=request):
                 data["response.error_code"] = argilla_error.code  # noqa
 
-        await self.track_data(topic=topic, data=data)
+        self.track_data(topic=topic, data=data)
+
+    def track_server_startup(self) -> None:
+        """
+        This method is used to track the launch of the server.
+
+        Returns:
+            None
+        """
+        topic = "startup"
+
+        self.track_data(topic=topic)
```

Comment:
```suggestion
        self.track_data(topic="startup")
```

I think you can do the same with the rest.